mkt/account/serializers.py

```diff
@@ -64,6 +64,8 @@ def validate(self, attrs):
         else:
             attrs['user'] = None
 
+        if attrs['feedback'].strip().__len__() < 1:
```

Comment:
Don't call `__len__` like that. Better call `len()` instead. But this is not even necessary, you can just do `if not attrs['feedback'].strip()`.